fe/fe-core/src/main/java/org/apache/doris/planner/RuntimeFilter.java

```diff
@@ -223,6 +227,20 @@ public static RuntimeFilter create(IdGenerator<RuntimeFilterId> idGen, Analyzer
         if (LOG.isTraceEnabled()) {
             LOG.trace("Generating runtime filter from predicate " + joinPredicate);
         }
+        if (ConnectContext.get().getSessionVariable().enableRemoveNoConjunctsRuntimeFilterPolicy) {
+            if (srcExpr instanceof SlotRef) {
+                if (!tupleHasConjuncts.contains(((SlotRef) srcExpr).getDesc().getParent().getId())) {
+                    // src tuple has no conjunct, don't create runtime filter
+                    return null;
+                } else {
+                    // runtime filter itself is a valid conjunct, add all the target tuple ids
+                    for (TupleId tupleId: targetSlots.keySet()) {
+                        tupleHasConjuncts.add(tupleId);
```

Comment:
I don't understand why `targetSlots::tupleID` is put into `tupleHasConjuncts`, can you explain, thx.

Reply:
I assume the targetSlots is where the runtime filter goes to.( if not, pls corret me ) In that case, the runtime filter can be treated as a normal conjunct. So if hash join node's build side is a scan node with no conjunct originally, but it has a runtime filter, we can also create a new runtime filter for it and push it to probe side.

Reply:
You are right, no problem.